json/3.0.0.wso2v2/pom.xml

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+ ~ Copyright (c) 2023, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
please update to the latest license header.

Reply:
updated